src/styles/normolize.css

```diff
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+}
```

Comment:
do not reset your margins with * selector, you should specify tags that you want to reset styles for
```suggestion
```